nexus-repository-helm/Dockerfile

```diff
@@ -3,23 +3,20 @@ ARG NEXUS_VERSION=3.18.0
 FROM maven:3-jdk-8-alpine AS build
 ARG NEXUS_VERSION=3.18.0
 ARG NEXUS_BUILD=01
-ARG HELM_VERSION=0.0.13
```

Comment:
Maybe I'm holding it wrong, but these changes to the Dockerfile don't appear to work for me.

The file needs to reference/update the helm plugin version.

Reply:
It might actually be easier to move `Dockerfile` back to the parent folder, and revert to the simpler version of that file. This would also remove any need to update the README.md.